src/embeddings/config/embedding.config.ts

```
@@ -0,0 +1,126 @@
+      litellm: {
+        model: process.env.LITELLM_EMBEDDING_MODEL || 'text-embedding-3-small',
+        baseUrl: process.env.LITELLM_BASE_URL,
+      },
+      openai: {
+        model: process.env.OPENAI_EMBEDDING_MODEL || 'text-embedding-3-small',
+        apiKey: process.env.OPENAI_API_KEY || '',
+        baseUrl: process.env.OPENAI_BASE_URL,
+      },
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)